content/docs/lists-and-keys.md

```diff
@@ -202,9 +212,9 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+Een goede vuistregel is dat de elementen binnen een `map()` aanroep keys nodig hebben.
```

Comment:
Als ik een suggestie mag doen: 
Een vuistregel is dat de elementen binnen de map() functie keys vereisen.
functie is nog niet helemaal goed vind ik zelf, maar operator is dan weer Engels.